include/ruby/intern.h

```diff
@@ -601,6 +601,7 @@ VALUE rb_io_print(int, const VALUE*, VALUE);
 VALUE rb_io_puts(int, const VALUE*, VALUE);
 VALUE rb_io_fdopen(int, int, const char*);
 VALUE rb_io_get_io(VALUE);
+VALUE rb_io_set_encoding(int, VALUE*, VALUE);
```

Comment:
I think it should be placed in `include/ruby/io.h`.
Other `rb_io` and `rb_file` functions have been declared here for a historical reason.

Reply:
Thanks, makes sense.